web/client/components/TOC/fragments/settings/FeatureInfo.jsx

```diff
@@ -60,7 +81,7 @@ export default class extends React.Component {
     render() {
         // the selected value if missing on that layer should be set to the general info format value and not the first one.
         const data = this.getInfoFormat(this.supportedInfoFormats());
-        return (
+        return this.state.loading ? <Loader /> : (
```

Comment:
right, i was a bit confused because i never saw it, now i understand why. tested by exporting/importing a map, opening layer properties yielded no request, going to the featureinfo tab did show the loader while getcapabilities were fetched.

applied in 0b00e8860